python/ray/tests/test_network_failures.py

```diff
@@ -0,0 +1,56 @@
+import os
+import pytest
+import sys
+
+import ray
+from ray._private.test_utils import Cgroup2NetworkBlocker
+from ray.exceptions import ActorUnavailableError
+
+pytest.skipif(sys.platform == "linux", "Cgroup2NetworkBlocker only works on Linux.")
+
+
+def test_network_blocker_works():
+    import requests
+
+    with Cgroup2NetworkBlocker(os.getpid()):
+        with pytest.raises(requests.exceptions.RequestException):
+            print(requests.get("http://example.com", timeout=5))
+    print(requests.get("http://example.com", timeout=5))
+
+
+@pytest.skipif(sys.platform == "linux", "Cgroup2NetworkBlocker only works on Linux.")
+def test_actor_unavailable(ray_start_cluster):
```

Comment:
Can you also add tests where incoming traffic is blocked, instead of outgoing traffic?